src/units/animation_component.hpp

```diff
@@ -134,4 +142,11 @@ class unit_animation_component
 
 	/** handle to the halo of this unit */
 	halo::handle unit_halo_;
+	/** handle to the abilities halos of this unit */
+	std::map<int, halo::handle> abil_halos_;
+	/** map used for check what each abil_halos_ don't modified */
```

Comment:
Without some more context, "don't modified" doesn't explain what it means.

Reply:
doxygen corrected